gatsby-node.esm.js

```diff
@@ -106,17 +110,20 @@ export const onPostBootstrap = async () => {
 
 const createRepositoryPages = (repositories, createPage) => {
   repositories.forEach(repository => {
-    const ownerName = repository.owner ? repository.owner.name : "";
-    const { name } = repository;
-    const repositoryPath = URLify(`${ownerName}/${name}`);
-    createPage({
-      path: repositoryPath,
-      component: path.resolve("./src/templates/repository/index.jsx"),
-      context: {
-        ownerName,
-        name,
-      },
-    });
+    const { archived } = repository;
+    if (!archived) {
```

Comment:
Since the repositories given to that function are from the query at line 164, they have already been filtered to exclude the archived repositories. This check is unnecessary.